relay-server/src/utils/rate_limits.rs

```diff
@@ -508,7 +569,6 @@ where
         summary.event_category = self.event_category.or(summary.event_category);
 
         let (enforcement, rate_limits) = self.execute(&summary, scoping)?;
-        envelope.retain_items(|item| self.retain_item(item, &enforcement));
```

Comment:
We should rename this function to something like `compute` to clarify that it does not enforce anything.